spec/components/webauthn_input_component_spec.rb

```diff
@@ -67,17 +85,6 @@
                 )
               end
             end
-
-            context 'with show_unsupported_passkey option true' do
```

Comment:
I wouldn't expect we'd remove this test?

Reply:
This was mistakenly removed. Fixed in [30687e2](https://github.com/18F/identity-idp/pull/11347/commits/30687e255ded28f04c386bb403afc73beb051d74).